snap/snapcraft.yaml

```diff
@@ -1,5 +1,5 @@
 name: snapcraft
-base: core22
+base: core24
```

Comment:
there is this lint warning:
```
- classic: usr/lib/x86_64-linux-gnu/librtmp.so.1: ELF rpath should be set to '$ORIGIN:/snap/core24/current/lib/x86_64-linux-gnu'.
```
it might be the same bug I bumped into with rockcraft: possibly two parts are staging this so file